src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i]);
+        }
```

Comment:
```suggestion
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
no need to store them